store/copr/coprocessor.go

```diff
@@ -706,14 +706,16 @@ func (worker *copIteratorWorker) handleTaskOnce(bo *Backoffer, task *copTask, ch
 	}
 
 	req := tikvrpc.NewReplicaReadRequest(task.cmdType, &copReq, options.GetTiKVReplicaReadType(worker.req.ReplicaRead), &worker.replicaReadSeed, kvrpcpb.Context{
-		IsolationLevel:   isolationLevelToPB(worker.req.IsolationLevel),
-		Priority:         priorityToPB(worker.req.Priority),
-		NotFillCache:     worker.req.NotFillCache,
-		RecordTimeStat:   true,
-		RecordScanStat:   true,
-		TaskId:           worker.req.TaskID,
-		ResourceGroupTag: worker.req.ResourceGroupTag,
+		IsolationLevel: isolationLevelToPB(worker.req.IsolationLevel),
+		Priority:       priorityToPB(worker.req.Priority),
+		NotFillCache:   worker.req.NotFillCache,
+		RecordTimeStat: true,
+		RecordScanStat: true,
+		TaskId:         worker.req.TaskID,
 	})
+	if worker.req.ResourceGroupTagger != nil {
+		worker.req.ResourceGroupTagger(req)
```

Comment:
Why cant client-go do it for us?

Reply:
My idea is to construct `ResourceGroupTag` where a specific request is created, such as `GetRequest`, `CommitRequest`, `coprocessor.Request`, etc. Most requests are constructed in `client-go`, but `coprocessor.Request` is constructed directly in TiDB. 
If we decide to completely fill the `ResourceGroupTag` in `client-go`, then we need to do special processing for `coprocessor.Request` and `coprocessor.BatchRequest` in the general `SendReqCtx` function, which seems unreasonable.